src/Symfony/Component/Serializer/Serializer.php

```diff
@@ -172,17 +172,17 @@ public function normalize($data, $format = null, array $context = [])
      *
      * @throws NotNormalizableValueException
      */
-    public function denormalize($data, $type, $format = null, array $context = [])
+    public function denormalize($data, $class, $format = null, array $context = [])
```

Comment:
💡 The parameter is called `$class` in the interface. This change synchronizes the method signatures.